vars/Debian_bookworm.yml

```diff
@@ -0,0 +1,18 @@
+---
+# The Python 2 package names
+python2_package_names:
+  # The python2 and python2-minimal packages do not exist on Debian
```

Comment:
So these packages were _removed_ from Debian Bookworm per the [`python2.7` news tracker](https://tracker.debian.org/news/1396456/python27-removed-from-testing/) (looks like it just happened actually) as I think part of finally removing Python 2 from Debian completely. The [`python2-rm` tracker](https://release.debian.org/transitions/html/python2-rm.html) is now returning a 404 so I'm guessing they're finished. I'm assuming with the eventual release of Debian Bookworm this file should become `Debian.yml` and the current `Debian.yml` should become `Debian_bullseye.yml`.

This is all to say maybe we should update the comment here to align with the fact that Python 2 has been removed from Debian Bookworm (and I would assume onward).

Reply:
Please see commit 1f788ff and #50.